test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -259,6 +266,12 @@ def expect_attribute_modifier(modifier_class, args)
     assert_response :success
   end
 
+  test "should update hostgroup_id of host" do
```

Comment:
Hard to believe such a simple thing failed, I had to run this manually first on 'develop' to check it. Wow!